app.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "watchmarket",
+  "buildpacks":[
+    {
+      "url":"heroku/go"
+    }
+  ],
+  "addons": [
+    "heroku-postgresql:hobby-dev",
+    "heroku-redis:hobby-dev"
+  ],
+  "environments": {
+    "review": {
+      "addons": [
+        {
+          "plan":"heroku-postgresql:hobby-dev",
+          "as":"STORAGE_POSTGRES"
```

Comment:
STORAGE_POSTGRES => STORAGE_POSTGRES_URL

Reply:
This is expected, `STORAGE_POSTGRES` will create `STORAGE_POSTGRES_` variables along with `STORAGE_POSTGRES_URL`